modules/ROOT/nav.adoc

```diff
@@ -13,193 +13,48 @@
- ** xref:guides:select.adoc[]
- ** xref:guides:join.adoc[]
- ** xref:guides:nest-unnest.adoc[]
- ** xref:guides:group-agg.adoc[]
-* xref:guides:indexes.adoc[]
- ** xref:guides:create-index.adoc[]
- ** xref:guides:place-index.adoc[]
- ** xref:guides:defer-index.adoc[]
- ** xref:guides:select-index.adoc[]
- ** xref:guides:drop-index.adoc[]
- ** xref:guides:index-advisor.adoc[]
-* xref:guides:manipulate.adoc[]
- ** xref:guides:insert.adoc[]
- ** xref:guides:update.adoc[]
- ** xref:guides:delete.adoc[]
-* xref:guides:transactions.adoc[]
-* xref:guides:import.adoc[]
-
-.Developer Information
-* xref:tutorials:couchbase-tutorial-student-records.adoc[Couchbase Tutorial]
+.Develop
+* xref:develop:intro.adoc[Developer's Intro]
+* xref:tutorials:couchbase-tutorial-student-records.adoc[Developer Tutorial]
   ** xref:tutorials:install-couchbase-server.adoc[]
   ** xref:tutorials:buckets-scopes-and-collections.adoc[]
   ** xref:tutorials:java-tutorial/install-couchbase-java-sdk.adoc[]
   ** xref:tutorials:java-tutorial/creating-the-students-collection.adoc[]
   ** xref:tutorials:java-tutorial/creating-the-courses-collection.adoc[]
   ** xref:tutorials:java-tutorial/retrieving-documents.adoc[]
   ** xref:tutorials:java-tutorial/adding-course-enrollments.adoc[]
-* xref:sdk:development-intro.adoc[Key-Value Operations]
+* xref:guides:intro.adoc[Developer Guides]
+  ** xref:guides:connect.adoc[]
+  ** xref:guides:kv-operations.adoc[]
+    *** xref:guides:creating-data.adoc[]
+    *** xref:guides:reading-data.adoc[]
+    *** xref:guides:updating-data.adoc[]
+    *** xref:guides:deleting-data.adoc[]
+    *** xref:guides:bulk-operations.adoc[]
+  ** xref:guides:query.adoc[]
+    *** xref:guides:select.adoc[]
+    *** xref:guides:join.adoc[]
+    *** xref:guides:nest-unnest.adoc[]
+    *** xref:guides:group-agg.adoc[]
+  ** xref:guides:indexes.adoc[]
+    *** xref:guides:create-index.adoc[]
+    *** xref:guides:place-index.adoc[]
+    *** xref:guides:defer-index.adoc[]
+    *** xref:guides:select-index.adoc[]
+    *** xref:guides:drop-index.adoc[]
+    *** xref:guides:index-advisor.adoc[]
+  ** xref:guides:manipulate.adoc[]
+    *** xref:guides:insert.adoc[]
+    *** xref:guides:update.adoc[]
+    *** xref:guides:delete.adoc[]
+  ** xref:guides:transactions.adoc[]
+  ** xref:guides:import.adoc[]
 * xref:home:ROOT:sdk.adoc[SDKs]
-** xref:sdk-extensions::distributed-acid-transactions.adoc[Distributed ACID Transactions]
-** xref:sdk-extensions::field-level-encryption.adoc[Field Level Encryption]
-** xref:sdk-extensions::response-time-observability.adoc[Response Time Observability]
-* xref:getting-started:starter-kits.adoc[Starter Kits]
+  ** xref:sdk-extensions::distributed-acid-transactions.adoc[Distributed ACID Transactions]
+  ** xref:sdk-extensions::field-level-encryption.adoc[Field Level Encryption]
+  ** xref:sdk-extensions::response-time-observability.adoc[Response Time Observability]
+  ** xref:sdk-extensions::spring-data-couchbase.adoc[Spring Data Couchbase]
```

Comment:
Was grouping starter kits under SDK deliberate?
In fact, should SDK be ** here?

Reply:
...and other bits below demoted a rank to bring them under Developer Guides?

Reply:
SDK is intentionally at the same level as the Developer Tutorial, Developer Guides, and the Services sections. The SDK Extensions topics are intentionally at the level below SDK. This is unchanged from the current situation, although I have indented the SDK Extensions topics to make it doubly obvious, and so that my editor can fold the list neatly 😉. 

I've added the Spring topic at the level below SDK alongside the other SDK Extensions.

I demoted the Starter Kits and placed it under SDK to try to keep the number of headings in the Develop section to 8 or fewer (as discussed). Seemed like a good place for it. Starter Kits is linked directly from the Developer Intro page, so should still be visible.

Reply:
Ah, think I got confused somewhere about levels in Server's navs (and my local build suffers from no lh nav in Server),
as to Starter Kits, I may not see this as being their natural home, but I have no useful alternative to suggest, so :shipit: